user-service/package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "dev": "nodemon index.js",
     "start": "node index.js",
-    "test": "set ENV=TEST && mocha --timeout 10000 --exit"

```

Comment:
Perhaps shouldn't remove since this ensures TEST environment database is used during testing